CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and simply didn't have the time to go back and retroactively create one.
 ### Changed
 - Changed session tracking so session IDs aren't reused
 - Changed zsh prompt to match CWD of other shell prompts
+- Improved exception handling in `Manager.interactive` ([#133](https://github.com/calebstewart/pwncat/issues/133))
```

Comment:
This need to be updated, the _fixes_ are no longer constrained to `Manager.interactive`

Reply:
Just pushed a fix which changes that message to be more generic. I don't think going into every single fix is productive in the change log. It just notes that we fixed a lot of exceptions/exception handler things...